### ISSUE TITLE
Setup CI / hosting of the oEQ-slideshow artifact

### ISSUE THREAD

**Issue by cbeach47** [openequella/openEQUELLA-slideshow#2]
Issue #1 is to stand up the app itself.  This issue is to hook in the CI for consistent builds, and a way to retrieve the artifact for simple inclusion in a web page.  

Webjars and github packages come to mind as options

At the least, a github actions CI flow that can release the artifacts as JS / CSS

**Comment by ChristianMurphy** [comment]
GitHub actions seems like a good starting point.

> way to retrieve the artifact for simple inclusion in a web page.

Does oEQ have a way to inject script into the webpage already?
If so aligning with and extending that mechanism could make sense.

> Webjars and github packages come to mind as options

For better or for worse `npm` has become the de-facto hosting platform for most javascript.
Tooling including: [webjars](https://www.webjars.org/), [unpkg](https://unpkg.com/), [skypack](https://www.skypack.dev/), [jspm](https://jspm.org/) mirror https://npmjs.com
If we can get the package on npm, all the above listed hosts (and some others unlisted) will automatically be able to distribute the code.

GitHub actions can publish to npm https://snyk.io/blog/github-actions-to-securely-publish-npm-packages (GitHub packages could also be published, it has npm compatible endpoints and publish mechanisms)

**Comment by cbeach47** [comment]
> Does oEQ have a way to inject script into the webpage already? If so aligning with and extending that mechanism could make sense.
I don't believe so.  This is meant to be dropped onto any web page (in or out of oEQ) and it should work (with the caveat of authentication), so while it'd be cool for it to be included eventually as a standard feature of oEQ and maybe pulled in with the build process, one of the core designs is that it can run outside of the app.

Let's go with github actions and npm for now - for the endpoint, I've used unpkg before with testing uP web components, and it has worked alright, so I'd be happy with GHA > NPM > unpkg unless you think there's a better / simpler way.
